spacy/pipeline/spancat.py

```diff
@@ -272,6 +272,24 @@ def predict(self, docs: Iterable[Doc]):
         scores = self.model.predict((docs, indices))  # type: ignore
         return indices, scores
 
+    def suggest_candidates(
+        self, docs: Iterable[Doc], spankey: str = "candidates"
+    ) -> List[Doc]:
+        """Use the spancat suggester to add a list of span candidates to a list of docs
+
+        docs (Iterable[Doc]): The documents to predict.
+        RETURNS: A list of Docs.
+
+        DOCS: WIP
```

Comment:
You can add the documentation in this PR and label it as being new since 3.3, then update the link here :-)

Reply:
How can I label the method? Is there an example that does something like this? :smile:

Reply:
Ah found it!